README.md

```diff
@@ -59,6 +60,13 @@ dataset.attrs["recoring_date"] = "2018-02-04"
 
 See the [documentation](https://exdir.readthedocs.io) for more information.
 
+## Benchmarks ##
+
+See [benchmarks.ipynb](tests/benchmarks/benchmarks.ipynb).
+
+A [live version](https://mybinder.org/v2/gh/CINPLA/exdir/dev?filepath=tests%2Fbenchmarks%2Fbenchmarks.ipynb)
```

Comment:
I am unable to get the binder working, I only get 404 : Not Found when binder has loaded the repository, but this can be because it is not merged?

Reply:
This is because the binder link uses the "dev" branch. It works if you change "dev" -> "dragly/performance". Should work when the PR is merged.